src/components/SearchBox.vue

```diff
@@ -2,14 +2,21 @@
   <div class="search-box-container">
     <div class="input-group search-input-container">
       <input type="text" class="form-control search-input" v-model.lazy="search"
-             placeholder="Search through biobanks and collections...">
+             placeholder="Search biobanks by name">
```

Comment:
could we use i18N to make this dynamic ?

Reply:
I think that if we do so, we should do it of everything.
We could always do it when ever a feature request for more configuration / internationalization comes along